### PR TITLE
Inject ads before headlines

### PR DESCRIPTION
**Pull request by brandonbk**

### CODE REVIEW

**Comment by brandonbk** [comment]
[Ad Insertion Logic: Avoid placement next to headings](https://trello.com/c/I73whgcs/22-ad-insertion-logic-avoid-placement-next-to-headings)

File: packages/marko-web-gam/browser/inject-ads.vue [hunk 1]
```diff
@@ -69,6 +69,8 @@ export default {
                   const $previous = $(this).prev();
                   if (component.detectEmbeds && $previous.attr('data-embed-type')) {
                     $(this).after(cleaned);
+                  } else if ($previous.is(':header')) {
```
Comment (zarathustra323) [inline packages/marko-web-gam/browser/inject-ads.vue]: Read the performance implications of using the jQuery `:header` pseudo-selector... https://api.jquery.com/header-selector/ 

I'll leave it up to you if you want to still use it, or use native selectors. I can go either way

File: packages/marko-web-gam/browser/inject-ads.vue [hunk 2]
```diff
@@ -67,7 +67,9 @@ export default {
               $child.nextAll(childSelector).each(function handleBefore() {
                 if ($(this).text().length > 1) {
                   const $previous = $(this).prev();
-                  if (component.detectEmbeds && $previous.attr('data-embed-type')) {
+                  if ($previous.is('h1,h2,h3,h4,h5,h6')) {
```
Comment (zarathustra323) [inline packages/marko-web-gam/browser/inject-ads.vue]: Make this header list a variable and re-use throughout

**Review by zarathustra323 — CHANGES_REQUESTED** [review]

File: packages/marko-web-gam/browser/inject-ads.vue (hunk 2 above)
Comment (brandonbk) [reply packages/marko-web-gam/browser/inject-ads.vue]: @zarathustra323 - done.

**Review by zarathustra323 — APPROVED** [review]

**Pull request merged**